apps/masterbots.ai/app/api/og/edge-client.ts

```
@@ -0,0 +1,56 @@
+          `,
+          variables: { threadId }
+        })
+      })
+      const data = await response.json()
+  
+      if (data.errors) {
+        throw new Error(data.errors[0].message)
+      }
+  
+      return data.data
+    } catch (error: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)